maxun-core/src/interpret.ts

```
@@ -45,6 +45,7 @@ interface InterpreterOptions {
   debugChannel: Partial<{
     activeId: Function,
     debugMessage: Function,
+    setActionType: Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)